bin/updateConfigFromEnv.js

```diff
@@ -0,0 +1,98 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+/* global require */
+
+const fs = require('fs');
+const path = require('path');
+const log = require('mozlog')('server');
+const url = require('url');
+
+const indexPath = path.join(__dirname, '..', 'dist', 'index.html');
+const META_TAG_PATTERN = /<meta name="fxa-oauth-console\/config\/environment" content="(.*)" \/>/g;
+
+function getConfigFromHtml(html) {
+  const match = META_TAG_PATTERN.exec(html)[1];
+  const decodedParsed = decodeURIComponent(match);
+  return JSON.parse(decodedParsed);
+}
+
+function getConfigsFromEnv() {
+  return {
+    oauthUri: process.env.OAUTH_URI || undefined,
```

Comment:
let's add some documentation into the README.md that list the available Docker env options. Also some instructions that show an example of running the container.